packages/realm-server/billing/billing_queries.ts

```diff
@@ -0,0 +1,314 @@
+import {
+  DBAdapter,
+  Expression,
+  addExplicitParens,
+  asExpressions,
+  param,
+  query,
+  separatedByCommas,
+} from '@cardstack/runtime-common';
+import { StripeEvent } from './stripe-webhook-handlers';
+
+export interface User {
+  id: string;
+  matrixUserId: string;
+  stripeCustomerId: string;
+}
+
+export interface Plan {
+  id: string;
+  name: string;
+  monthlyPrice: number;
+  creditsIncluded: number;
+}
+
+export interface Subscription {
+  id: string;
+  userId: string;
+  planId: string;
+  startedAt: number;
+  endedAt?: number;
+  status: string;
+  stripeSubscriptionId: string;
+}
+
+export interface SubscriptionCycle {
+  id: string;
+  subscriptionId: string;
+  periodStart: number;
+  periodEnd: number;
+}
+
+export interface LedgerEntry {
+  id: string;
+  userId: string;
+  creditAmount: number;
+  creditType: string;
+  subscriptionCycleId: string;
+}
+
+export async function insertStripeEvent(
+  dbAdapter: DBAdapter,
+  event: StripeEvent,
+) {
+  let { valueExpressions, nameExpressions: _nameExpressions } = asExpressions({
+    stripe_event_id: event.id,
+    event_type: event.type,
+    event_data: event.data,
+  });
+  await query(dbAdapter, [
+    `INSERT INTO stripe_events (stripe_event_id, event_type, event_data) VALUES`,
```

Comment:
you can actually use the `nameExpressions` instead of having to enter the column names--that's the whole point of that variable